examples/contrived.py

```diff
@@ -0,0 +1,22 @@
+import instant_segment
+
+
+def main():
+    unigrams = []
+    unigrams.append(("choose", 50))
```

Comment:
How about we use some "real" weights here to make it slightly less contrived? Something like

- unigrams: choose 80_000, chooses 7_000, spain 20_000, pain 90_000
- bigrams: (`choose`, `spain`) 7, (`chooses`, `pain`) 0

With the current weights, I don't like that the bigram weights in particular are the same, which I fear might make it less clear what is happening under the covers.

Reply:
I like it